src/coreclr/src/vm/eventpipe.cpp

```diff
@@ -94,12 +96,121 @@ void EventPipe::Initialize()
 #endif
     }
 
-
     {
         CrstHolder _crst(GetLock());
         if (tracingInitialized)
             s_state = EventPipeState::Initialized;
     }
+
+    EnableViaEnvironmentVariables();
+}
+
+//
+// If EventPipe environment variables are specified, parse them and start a session
+// 
+void EventPipe::EnableViaEnvironmentVariables()
+{
+    STANDARD_VM_CONTRACT;
+    if (CLRConfig::GetConfigValue(CLRConfig::INTERNAL_EnableEventPipe) != 0)
+    {
+        LPWSTR eventpipeConfig = NULL;
+        CLRConfig::GetConfigValue(CLRConfig::INTERNAL_EventPipeConfig, &eventpipeConfig);
+        LPCWSTR eventpipeOutputPath = Configuration::GetKnobStringValue(W("EventPipeOutputPath"));
```

Comment:
Same as above assuming `Configuration::GetKnobStringValue` is calling the `CLRConfig` logic internally.